src/app/components/client/ExposureCard.tsx

```diff
@@ -316,26 +429,30 @@ export const ExposureCard = (props: ExposureCardProps) => {
             </dt>
             <dd>
               <span className={styles.exposureCompanyTitle}>
-                {exposureName}
+                {subscriberBreach.title}
               </span>
             </dd>
             <dt className={styles.visuallyHidden}>
               {l10n.getString("exposure-card-exposure-type")}
             </dt>
             <dd className={styles.hideOnMobile}>
-              <ExposureTypeEl type={exposureData} />
+              {l10n.getString("exposure-card-exposure-type-data-breach")}
             </dd>
             <dt className={styles.visuallyHidden}>
               {l10n.getString("exposure-card-date-found")}
             </dt>
             <dd className={styles.hideOnMobile}>
-              {dateFormatter.format(props.dateFound)}
+              {dateFormatter.format(
+                // We should be able to result that HIBP's `addedDate` property is a properly-formatted data string
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                parseIso8601Datetime(subscriberBreach.addedDate)!
```

Comment:
Heh, that was a fun one, well spotted. So apparently, `SubscriberBreach.addedDate`, at least in this case, is actually a `Date` object rather than an ISO 8601 date string (and likewise for `modifiedDate` and `breachDate`). The previous date formatted just passed it into the `Date` constructor, which then just returned that same date, but `parseIso8601DatetimeString` actually checks if it's a string, and returns `null` if not.

So I just submitted https://github.com/mozilla/blurts-server/pull/3369 that should ensure it is consistently a `Date` object. Once that is merged, I can update this PR to not call `parseIso8601DatetimeString` for `SubscriberBreach`es. (And TypeScript will prevent me from merging this until I do, so I won't forget :) )